pkg/volume/testing/testing.go

```diff
@@ -416,7 +415,9 @@ func (plugin *FakeVolumePlugin) getFakeVolume(list *[]*FakeVolume) *FakeVolume {
 	volume.VolumesAttached = make(map[string]types.NodeName)
 	volume.DeviceMountState = make(map[string]string)
 	volume.VolumeMountState = make(map[string]string)
-	*list = append(*list, volume)
+	if list != nil {
```

Comment:
I've checked the callers and a `nil` pointer is never passed to this method. Both checks can be just removed to avoid any confusion.

Reply:
@ash2k , so in that situation, we do not need this PR. closing it.

Reply:
@gaurav1086 I think it'd be an improvement to remove the `list != nil` check on line 404 because it causes confusion. It confused us, didn't it?

Reply:
@ash2k , I agree. I think its a good practice to have a non-nil/null check regardless. Thoughts ?